libraries/GCS_MAVLink/GCS.h

```diff
@@ -323,6 +324,8 @@ class GCS_MAVLINK
     // this allows us to detect the user wanting the CLI to start
     uint8_t        crlf_count;
 
+    static bool _in_update;
```

Comment:
Making it static was me being lazy... It's not worse than it current it is due to the other variables used on this function being static